.pipelines/ci.yml

```diff
@@ -15,29 +15,28 @@ jobs:
   #######
   # Linux
   #######
-
   - job: Linux
     pool:
       vmImage: 'ubuntu-latest'
 
     strategy:
       matrix:
-        py310-1121:
+        py310-1131:
           python.version: '3.10'
+          torch.version: 'torch torchvision torchaudio'
+          ort.version: '1.13.1'
+        py39-1121:
+          python.version: '3.9'
           torch.version: 'torch==1.12.1+cpu torchvision==0.13.1+cpu torchaudio==0.12.1+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
           ort.version: '1.12.0'
```

Comment:
What's the reason for the ORT version differing? Does it only affect the CI build or will it affect the ort-ext python package dependencies.